doc/migrating-configs.md

```diff
@@ -2,6 +2,41 @@
 
 Occasionally, there are changes made to Ignition's configuration that break backward compatibility. While this is not a concern for running machines (since Ignition only runs one time during first boot), it is a concern for those who maintain configuration files. This document serves to detail each of the breaking changes and tries to provide some reasoning for the change. This does not cover all of the changes to the spec - just those that need to be considered when migrating from one version to the next.
 
+## From Version 2.2.0 to 2.3.0
+
+There are not any breaking changes between versions 2.2.0 and versions 2.3.0 of the configuration specification. Any valid 2.2.0 configuration can be updated to a 2.3.0 configuration by simply changing the version string in the config.
+
+The 2.3.0 version of the configuration is greatly improved over version 2.2.0, with many new fields and behaviors added to the specification.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### More expressive partitioning
+
+The `disks` section gained support for more complex partitioning operations. The `partitions` field has gained two new fields: `wipePartitionEntry` and `shouldExist`. The former indicates that Ignition is allowed to wipe a partition entry from the disk's partition table if necessary to satisfy the config. The latter indicates whether a partition with the specified number should exist on the disk. See the [Operator Notes](https://coreos.com/ignition/docs/latest/operator-notes.html) for more details.
+
+This allows for existing partitions to be resized (when possible) and for partitions to be deleted. 
+
+The `start` and `size` fields which specified the starting sector and size of a partition (in sectors) are deprecated in favor of `startMB` and `sizeMB` which are specified in megabytes. Specifying 0 for the new fields has the same meaning as specifying 0 for the old fields.
```

Comment:
I was cross-checking with `sgdisk --new` [manpage](https://linux.die.net/man/8/sgdisk) and I fear that those are actually mebibytes (i.e. 2^20 instead of 10^6). If that is the case, field name is misleading and docs are wrong.

However there are tests covering this and they seem correct at a first glance, so I am quite confused now. Can you please double check that?

Reply:
Fixed

Reply:
Description and keywords need to be adjusted after https://github.com/coreos/ignition/pull/660.

Reply:
Fixed